rust_src/src/remacs_sys.rs

```diff
@@ -1377,6 +1377,7 @@ extern "C" {
     pub static initial_obarray: Lisp_Object;
     pub static lispsym: Lisp_Symbol;
     pub static minibuf_level: EmacsInt;
+    pub static minibuf_prompt: Lisp_Object;
```

Comment:
Not static any more....

Reply:
I think @shaleh is referring to the fact that you've removed the `static` in minibuf.c.

Reply:
my understanding was that `static` in C was mostly for scoping. `minibuf_prompt` is laced through `minibuf.c`. at first, i had a `get_minibuf_prompt` function to make up for the fact, but that seemed like more of a hack than removing the `static` keyword.